### ISSUE TITLE
Prevent user nobody deletion

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#4723]
Modify /lua/admin/users.lua to prevent user nobody to be deleted

This ticket depends on #4722

**Comment by simonemainardi** [comment]
This is no longer needed as now a REST API token has been implemented on a per-user basis. Users can generate a token and use it for REST API queries.
